charts/operator/Chart.yaml

```diff
@@ -34,9 +34,9 @@ annotations:
       url: https://helm.sh/docs/intro/install/
   artifacthub.io/images: |
     - name: redpanda-operator
-      image: docker.redpanda.com/redpandadata/redpanda-operator:v23.2.5
```

Comment:
should we use 23.2.8 instead?

Reply:
I wanted to merge this PR to run nightly and create PR from that run.